README.md

```diff
@@ -1,3 +1,52 @@
-# CloudEvents Plugin for Jenkins
+# CloudEvents-Plugin
+
+## Introduction
+
+The CloudEvents Plugin for Jenkins allows interoperability between different CI/CD tools by adopting industry-standard specification for events called [CloudEvents](https://cloudevents.io/). By using this plugin in your workflows, you can emit and consume CloudEvents for various events in Jenkins relating to Jenkins objects (jobs, projects, queues, etc).
+
+## Getting started
+
+There are 2 usages of this plugin:
+1. **Jenkins as a Source.**
+   
+    1. To configure Jenkins as a Source, first click on *Manage Jenkins* in the Root-Actions menu on the left.
+        ![Manage Jenkins](public/manage_jenkins.png)
+       
+    2. Once you are in Manage Jenkins UI, search for *Configure System* under *System Configuration*. 
+        ![Configure System](public/configure_system.png)
+       
+    3.  Inside the *Configure System* UI, scroll down to the *CloudEvents Plugin* section. 
+        Here, each block represents an endpoint, which comprises a valid Sink-URL and type of event
+        that URL is configured to receive.
+        - Select HTTP Sink URL.
+        - Select an event from the dropdown which will be sent to the sink you have configured.
+        
+        ![Configure Endpoints](public/add_endpoints.png)
+          
+    4. You can also add more endpoints and configure them similarly. 
+        This way, each endpoint can be configured to receive specific events.
+       ![Add More Endpoints](public/add_more.png)
+
+2. **Jenkins as a Sink.**
+    1. To use Jenkins as a Sink, simply enter the Jenkins Sink URL as the Sink to send CloudEvents from other systems.
```

Comment:
Please do not use the words "simply" or "easy" or any other word descirbing to the difficulty of the task being explained in the docs.
Refer to https://justsimply.dev/

Reply:
oh, that makes sense. I'll take care of this.